Implement escape feature (#7)
DiscordBot-PMMP/ChatBridge: resources/help/english.txt

      - "12345678912345678"                             <- Discord channel IDs
      - "42356325325312412"

>     escapes:                                            <- List of characters to escape "{MESSAGE}", to disable, delete the "- @" line and replace it with "escapes: []".
>       - "@"                                             <- A space is added after every '@'

Author: ```suggestion
    escapes: ["@"]                                      <- List of characters to escape {MESSAGE}, with a space inserted after the "@" by default.
```

>     escapes:                                             <- List of characters to escape "{MESSAGE}", to disable, delete the "- @" line and replace it with "escapes: []".
>       - "@"                                              <- A space is added after every '@'

Author: ```suggestion
    escapes: ["@"]                                      <- List of characters to escape {MESSAGE}, with a space inserted after the "@" by default.
```
